### DISCUSSION TITLE
Hide Computers Tab

### DISCUSSION THREAD

**Discussion by dnauman1** [lithnet/access-manager#206] [Ideas]
Is there a way to hide the option for Computers from the home screen and only show Roles? I only want to use Lithnet to provide JIT access by adding users to a specified group. I have it configured to do so but I want the home screen to default to Roles and not show the option for Computers.

**Comment by ryannewington** [comment]
@dnauman1 

There isn't a way to do this today, but it's a great idea and we'll add it to the backlog for a future release.
